config/main.py

```diff
@@ -1999,6 +2002,20 @@ def override_config_db(config_db, config_input):
     click.echo("Overriding completed. No service is restarted.")
 
 
+def yang_hard_dependency_check(config_json):
+    aaa_yang_hard_dependency_check(config_json)
+
+
+def aaa_yang_hard_dependency_check(config_json):
+    AAA_TABLE = config_json.get("AAA", {})
+    TACPLUS_TABLE = config_json.get("TACPLUS", {})
+    tacacs_enable = "tacacs+" in AAA_TABLE.get("authorization", {}).get("login", "")
```

Comment:
>in [](http://example.com/codeflow?start=30&length=2)

It would be cautious to split the "login" value, and check match one of the item.

Reply:
resolved